adafruit_displayio_ssd1306.py

```diff
@@ -78,3 +78,33 @@ def __init__(self, bus, **kwargs):
             brightness_command=0x81,
             single_byte_bounds=True,
         )
+        self._is_awake = True  # Display starts in active state (_INIT_SEQUENCE)
+
+    @property
+    def is_awake(self):
+        """
+        The power state of the display. (read-only)
+
+        True if the display is active, False if in sleep mode.
+        """
+        return self._is_awake
+
+    def sleep(self):
```

Comment:
should be part of the `awake` property = setting it to True or False puts it to sleep or not

Reply:
We already have a sleep API merged in for the SH1107 so let's leave this as-is so it's consistent: https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_SH1107/pull/2/files#diff-31026eacc758006ff27f43d9fff4ad747cdfd61cbd17dcb69e7e43eac020de2cR75